lib/amorail/entities/contact.rb

```diff
@@ -10,21 +10,21 @@ def request_attributes
           contacts: {
             add: [
               {
-                name: self.name,
-                linked_leads_id: [self.linked_leads_id],
-                company_name: self.company_name,
+                name: name,
+                linked_leads_id: [linked_leads_id],
+                company_name: company_name,
                 custom_fields: [
                   {
-                    id: 1460587,
-                    values: [{value: self.job_position}]
+                    id: properties.contact.position.id,
+                    values: [{value: job_position}]
                   },
                   {
-                    id: 1460589,
-                    values: [{value: self.phone, enum: 'MOB'}]
+                    id: properties.contact.phone.id,
+                    values: [{value: phone, enum: 'MOB'}]
                   },
                   {
-                    id: 1460591,
-                    values: [{value: self.email, enum: 'WORK'}]
+                    id: properties.contact.email.id,
+                    values: [{value: email, enum: 'WORK'}]
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.